pywinauto/timings.py

```diff
@@ -297,40 +330,58 @@ def WaitUntil(
      
     """
     
-    start = time.time()
-
-    func_val = func(*args)
-    # while the function hasn't returned what we are waiting for    
-    while not op(func_val, value):
-            
-        # find out how much of the time is left
-        time_left = timeout - ( time.time() - start)
-    
-        # if we have to wait some more        
-        if time_left > 0:
-            # wait either the retry_interval or else the amount of
-            # time until the timeout expires (whichever is less)
-            time.sleep(min(retry_interval, time_left))
-            func_val = func(*args)
-        else:
-            err = TimeoutError("timed out")
-            err.function_value = func_val
-            raise err
-            
-    return func_val
-
-
+    return always_wait_until(timeout, retry_interval, value, op)(func)(*args)
+    	
+    WaitUntil = wait_until 
```

Comment:
@AnufrievDE it seems that you put ```WaitUntil = wait_until``` and ```WaitUntilPasses = wait_until_passes``` under a wrong scope. These should be global variables. This is why the unit tests failed.